src/Styleguide/Containers/Artist/AuctionResults/AuctionResults.tsx

```diff
@@ -0,0 +1,236 @@
+import * as React from "react"
+import { createRefetchContainer, graphql, RelayRefetchProp } from "react-relay"
+
+import { Subscribe } from "unstated"
+import { FilterState } from "./state"
+
+import { AuctionResults_artist } from "__generated__/AuctionResults_artist.graphql"
+
+import { AuctionResultItem } from "Styleguide/Components/AuctionResultItem"
+import { Pagination } from "Styleguide/Components/Pagination"
+import { Select } from "Styleguide/Elements/Select"
+
+interface Props {
+  relay: RelayRefetchProp
+  artist: AuctionResults_artist
+}
+
+const PAGE_SIZE = 20
+
+const Sorts = [
+  {
+    value: "DATE_DESC",
+    text: "Most Recent",
+  },
+  {
+    value: "ESTIMATE_AND_DATE_DESC",
+    text: "Estimate",
+  },
+  {
+    value: "PRICE_AND_DATE_DESC",
+    text: "Sale Price",
+  },
+]
+
+class AuctionResults extends React.Component<Props> {
+  loadPrev = () => {
+    const {
+      startCursor,
+      hasPreviousPage,
+    } = this.props.artist.auctionResults.pageInfo
+    if (hasPreviousPage) {
+      this.loadBefore(startCursor)
+    }
+  }
+  loadNext = () => {
+    const { hasNextPage, endCursor } = this.props.artist.auctionResults.pageInfo
+    if (hasNextPage) {
+      this.loadAfter(endCursor)
+    }
+  }
+  loadBefore(cursor) {
+    this.props.relay.refetch(
+      {
+        first: null,
+        before: cursor,
+        artistID: this.props.artist.id,
+        after: null,
+        last: PAGE_SIZE,
+      },
+      null,
+      error => {
+        if (error) {
+          console.error(error)
+        }
+      }
+    )
+  }
+  loadAfter = cursor => {
+    this.props.relay.refetch(
+      {
+        first: PAGE_SIZE,
+        after: cursor,
+        artistID: this.props.artist.id,
+        before: null,
+        last: null,
+      },
+      null,
+      error => {
+        if (error) {
+          console.error(error)
+        }
+      }
+    )
+  }
+
+  renderPagination() {
+    return (
+      <div>
+        <Pagination
+          first={this.props.artist.auctionResults.pageCursors.first}
+          last={this.props.artist.auctionResults.pageCursors.last}
+          around={this.props.artist.auctionResults.pageCursors.around}
```

Comment:
These 3 lines can be replaced with: `{...this.props.artist.auctionResults.pageCursors}`

Reply:
oh yea cool! i finally got comfortable/started using `const { var, ...notVars } = options` !